freqtrade/rpc/telegram.py

```diff
@@ -193,7 +193,7 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
             else:
                 msg['stake_amount_fiat'] = 0
 
-            message = (f"\N{LARGE BLUE CIRCLE} *{msg['exchange']}:* Buying {msg['pair']}\n"
+            message = (f"\N{LARGE BLUE CIRCLE} *{msg['exchange']}:* Buying {msg['pair']} (#{msg['trade_id']})\n"
```

Comment:
This line becomes too long (making CI fail with flake8) ... simply wrap the new stuff around into a new line ... ;)